pkg/operator/ceph/file/filesystem.go

```diff
@@ -46,8 +48,8 @@ func createFilesystem(
 		for _, p := range fs.Spec.DataPools {
 			dataPools = append(dataPools, p.ToModel(""))
 		}
-		f := mdsdaemon.NewFS(fs.Name, fs.Spec.MetadataPool.ToModel(""), dataPools, fs.Spec.MetadataServer.ActiveCount)
-		if err := f.CreateFilesystem(context, fs.Namespace); err != nil {
+		f := newFS(fs.Name, fs.Spec.MetadataPool.ToModel(""), dataPools, fs.Spec.MetadataServer.ActiveCount)
+		if err := f.doFilesystemCreate(context, fs.Namespace); err != nil {
```

Comment:
why not just `f.createFilesystem()`?

Reply:
There is already a `createFilesystem` method which I believe is the method which calls this. There has to be some method of differentiation, and this seemed like it was the most straightforward while still keeping the same wording as previously.